taxonomy/models.py

```diff
@@ -1042,3 +1042,43 @@ class Meta:
         app_label = 'taxonomy'
         verbose_name = 'Industry'
         verbose_name_plural = 'Industries'
+
+
+class B2CJobAllowList(models.Model):
+    """
+    Model for storing admin configuration for B2C Job Allowlist entries
+
+    .. no_pii:
+    """
+    external_id = models.CharField(
+        max_length=255,
+        blank=False,
+        null=False,
+        unique=True,
+        help_text=_(
+            'The external identifier for the job received from API.'
```

Comment:
I had this half-baked assumption that this model field would be a one-to-one relationship to a Job. Any benefit to not having a relationship to the other model?

Reply:
I was going to ask you about that! I think I was being to literal about mapping the config. I can bake that more fully. Is there any problem with an update making a job go away, and a mapping to a job blowing up?

Reply:
Done. It makes more sense now.